gRPC/Client Examples/EvokedPotentialsApp/BICManager.cs

```diff
@@ -168,6 +175,43 @@ public bool BICConnect()
             Console.WriteLine("Connecting to implantable device.");
             var connectDeviceReply = deviceClient.ConnectDevice(new ConnectDeviceRequest() { DeviceAddress = DeviceName, LogFileName = "./deviceLog.txt" });
 
+            // Print out impedances of electrodes
```

Comment:
Seems like we should've had a separate impedance check PR instead of bundling with reconnect?

Reply:
Makes sense, impedance check changes will be moved to a separate PR